html/console.html

```diff
@@ -18,7 +18,17 @@
           <option value="19200">19200</option>
           <option value="9600">9600</option>
         </select>
-        &nbsp; Fmt: 8N1
+        &nbsp; Fmt:
+	<select id="fmt-sel" class="pure-button" href="#">
+	  <option value="8N1">8N1</option>
+	  <option value="8E1">8E1</option>
+	  <option value="8N2">8N2</option>
+	  <option value="8E2">8E2</option>
+	  <option value="7N1">7N1</option>
+	  <option value="7E1">7E1</option>
+	  <option value="7N2">7N2</option>
+	  <option value="7E2">7E2</option>
+	</select>
```

Comment:
I just push it over to my github repository.
https://github.com/bitdump/esp-link/tree/add-uart-format-support-v2
 Now it loads the flash config into uart init.